Assets/Plugins/Source/Core/IPlatformSpecifics.cs

```diff
@@ -79,7 +79,7 @@ public interface IEOSCreateOptions
         Utf8String CacheDirectory { get; set; }
         uint TickBudgetInMilliseconds { get; set; }
 
-#if !(UNITY_STANDALONE_WIN || UNITY_EDITOR_WIN || UNITY_STANDALONE_LINUX|| UNITY_EDITOR_LINUX)
+#if !(UNITY_STANDALONE_WIN || UNITY_EDITOR_WIN || UNITY_STANDALONE_LINUX || UNITY_EDITOR_LINUX || UNITY_STANDALONE_OSX || UNITY_EDITOR_OSX)
         Epic.OnlineServices.IntegratedPlatform.IntegratedPlatformOptionsContainer IntegratedPlatformOptionsContainerHandle { get; set; }
 #endif
```

Comment:
I know this might seem super pedantic - but couldn't / shouldn't we change this to the more easily read:

```C#
#if !(UNITY_STANDALONE || UNITY_EDITOR)
```

?

Same question applies to the other file that's been changed in this PR.

Reply:
If it doesn't affect mobile platforms then should be squashable.